alfalfa_worker/Dockerfile

```diff
@@ -1,11 +1,15 @@
-FROM michaelwetter/ubuntu-1804_jmodelica_trunk AS base
+FROM ubuntu:18.04 AS base
```

Comment:
This whole file looks great, if we are worried about size, which I think is a secondary consideration, then we could collapse a lot of these calls and only save what matters.